From 78c5335b1aa6a2ba60dbad7e5bf10b3ef517c18a Mon Sep 17 00:00:00 2001
Message-Id: <78c5335b1aa6a2ba60dbad7e5bf10b3ef517c18a.1649937120.git.geliang.tang@suse.com>
From: Randy Dunlap <rdunlap@infradead.org>
Date: Wed, 22 Dec 2021 16:48:02 -0800
Subject: [PATCH] ntb_hw_switchtec: fix the spelling of "its"
Git-commit: 78c5335b1aa6a2ba60dbad7e5bf10b3ef517c18a
Patch-mainline: v5.17-rc1
References: bsn#170
Modified-by-SEL: No


Use the possessive "its" instead of the contraction "it's" (it is)
in user messages.

Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Cc: Kurt Schwemmer <kurt.schwemmer@microsemi.com>
Reviewed-by: Logan Gunthorpe <logang@deltatee.com>
Signed-off-by: Jon Mason <jdmason@kudzu.us>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 drivers/ntb/hw/mscc/ntb_hw_switchtec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/ntb/hw/mscc/ntb_hw_switchtec.c b/drivers/ntb/hw/mscc/ntb_hw_switchtec.c
index 4c6eb61a6ac6..b2d956c2d610 100644
--- a/drivers/ntb/hw/mscc/ntb_hw_switchtec.c
+++ b/drivers/ntb/hw/mscc/ntb_hw_switchtec.c
@@ -297,7 +297,7 @@ static int switchtec_ntb_mw_set_trans(struct ntb_dev *ntb, int pidx, int widx,
 		 * (see CMA_CONFIG_ALIGNMENT)
 		 */
 		dev_err(&sndev->stdev->dev,
-			"ERROR: Memory window address is not aligned to it's size!\n");
+			"ERROR: Memory window address is not aligned to its size!\n");
 		return -EINVAL;
 	}
 
-- 
2.34.1

